datastore/keycompat_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package datastore
+
+import (
+	"reflect"
+	"testing"
+)
+
+var testCasesKeyCompat = []struct {
```

Comment:
Not a big deal, but I usually prefer putting the test cases directly in the test where they're used. Then the slice can just be named `tests` because it's clear what they're for.

Reply:
Done